src/common/Calendar.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
-  import { lpad } from "@/util/string";
   import { _, date, time } from "svelte-i18n";
+  import { lpad } from "@/util/string.js";
```

Comment:
Insert generic `lpad` joke here

Reply:
Really resisted the urge to take that out now that [`padStart()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) exists. But there's actually a bunch of things like that and I'll get to it eventually.